frontend/src/components/pages/teacher/DisplayClassroomPage/index.tsx

```diff
@@ -176,14 +176,14 @@ const DisplayClassroomsPage = () => {
         )}
         {!loading && (
           <IconButton
-            aria-label="Edit classroom"
-            color="blue.300"
+            aria-label="edit-classroom"
             icon={<EditOutlineIcon />}
-            minW={0}
             onClick={() => {
               classroomFormMethods.reset();
               onClassroomModalOpen();
             }}
+            size="icon"
+            variant="icon"
```

Comment:
What's the purpose of the extra variant here, out of curiosity?

Reply:
Icon button gets its theming from the `button.ts` file (i.e under the hood it's actually a button). So we can't style the default variant.

Reply:
Gotcha, it's fine as is then. We could potentially make a wrapper component just to specify the default variant and size, but probably not worth it then.